javaslang/src/main/java/javaslang/collection/Vector.java

```diff
@@ -882,12 +848,12 @@ public int length() {
     @Override
     public Tuple2<Vector<T>, Vector<T>> partition(Predicate<? super T> predicate) {
         Objects.requireNonNull(predicate, "predicate is null");
-        final java.util.List<T> left = new ArrayList<>(), right = new ArrayList<>();
+        final ArrayList<T> left = new ArrayList<>(), right = new ArrayList<>();
```

Comment:
Did you change it to ArrayList because of the full qualified name? It is ok for me here because it is internal.